pkg/apis/config/v1alpha1/mesh_config.go

```diff
@@ -172,10 +172,10 @@ type MeshConfigList struct {
 // FeatureFlags is a type to represent OSM's feature flags.
 type FeatureFlags struct {
```

Comment:
Should we remove the omit empty from all the bool values in this and not just WASM and Egress ?

Reply:
I removed `omitempty` for all boolean fields whose default value is true in MeshConfig. We can also do it for other CRDs. But with this approach, developers meed to care about whether `omitempty` should be added, which is error prune.